web/src/components/LocationToolbar.tsx

```diff
@@ -7,14 +7,15 @@ type PropsType = {
   openFeedbackModal: (rating: FeedbackRatingType) => void
   children?: ReactNode
   viewportSmall: boolean
+  iconDirection?: 'row' | 'column'
 }
 
 class LocationToolbar extends React.PureComponent<PropsType> {
   render(): ReactNode {
-    const { viewportSmall, children, openFeedbackModal } = this.props
+    const { viewportSmall, children, openFeedbackModal, iconDirection = 'column' } = this.props
```

Comment:
no need to set the default value here, as it is set to "column" in `Toolbar.tsx` anyway if not set. I think it is better not to set the default twice.